rllib/execution/replay_ops.py

```diff
@@ -1,136 +1,10 @@
-from typing import List, Any, Optional
+from typing import List, Optional
```

Comment:
we should be able to delete this whole file?
we should be using Arthur's RBs now right?

Reply:
Let me get rid of it..